packages/prisma/seed-app-store.ts

```diff
@@ -205,6 +207,8 @@ export default async function main() {
       client_secret: process.env.HUBSPOT_CLIENT_SECRET,
     });
   }
+  // No need to check if environment variable is present, the API Key is set up by the user, not the system
```

Comment:
WDYM with this? How is the key set by the user? Do we have a UI for this? Is it saved a credential? If not then this is a system key and should be saved in the DB for consistency with every other app.

Reply:
Exactly, the setup page for the app gets the key from the user, so in terms of validating if the app can show up as installable, there is no need to check here for any environment variable, as the environment variable for the Close.com API Key is just for our own usage.